### DISCUSSION TITLE
How do I style the editor-content?

### DISCUSSION THREAD

**Discussion by jezsung** [ueberdosis/tiptap#848] [General Questions] [answered]
I tried to style the editor-content with the following scss

``` vue
<style lang="scss" scoped>
::v-deep .ProseMirror {
  height: 100vh;
  border-style: none;
}

::v-deep .ProseMirror-focused {
  border-style: none;
}
</style>
```
`::v-deep .ProseMirror {...}` part is applied properly but `::v-deep .ProseMirror-focused {}` doesn't work. I want to remove the border-style when it gets focused. How do I do that?

**Comment by andreasvirkus** [comment, marked as answer]
@jezsung I believe you're looking for
```
::v-deep .ProseMirror:focus {
  outline: none;
}
```

**Note** some sort of visual indicator of focus on an input element is recommended to maintain better accessibility for your webpage/-app
